src/Client.php

```diff
@@ -344,12 +344,16 @@ public function contentEndpointRequest(string $endpoint, array $arguments, $body
         return $response;
     }
 
-    public function rpcEndpointRequest(string $endpoint, array $parameters): array
+    public function rpcEndpointRequest(string $endpoint, array $parameters = null)
```

Comment:
Changing the signature of a method like this is a breaking change. Could you please add back the return type hint. 

Just return an empty array if the response of dropbox is empty.